x/intertx/keeper/keeper.go

```diff
@@ -0,0 +1,42 @@
+package keeper
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	capabilitytypes "github.com/cosmos/cosmos-sdk/x/capability/types"
+	"github.com/tendermint/tendermint/libs/log"
+
+	"github.com/regen-network/regen-ledger/x/intertx"
+)
+
+type Keeper struct {
+	cdc codec.BinaryCodec
+
+	storeKey storetypes.StoreKey
```

Comment:
Is `storeKey` is required? I don't find its usage.

Reply:
hmm, actually not used anywhere. removed here https://github.com/regen-network/regen-ledger/pull/1477/commits/e30dea39eb0d613dd02a4e9a85fc824b6101cdc0